src/Processors/CsvProcessor.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace R64\ContentImport\Processors;
+
+use Illuminate\Support\Facades\Storage;
+use League\Csv\Reader;
+
+class CsvProcessor implements FileProcessorContract
+{
+    public function read(string $path)
+    {
+        $stream = fopen(Storage::disk('local')->path($path), 'r');
+        $csv = Reader::createFromStream($stream);
+
+        if (config('content_import.heading_row', true)) {
```

Comment:
If we'll resolve this from container, this kind of config options should be injected through constructor.